Lib/fontTools/feaLib/builder.py

```diff
@@ -1154,7 +1159,7 @@ def makeOpenTypeAnchor(anchor):
 def makeOpenTypeValueRecord(v, pairPosContext):
     """ast.ValueRecord --> (otBase.ValueRecord, int ValueFormat)"""
```

Comment:
the docstring doesn't match the return value (but you probably know that, and deliberately split the doc fixes to a different PRs). Ideally each PR is coherent within itself, so it's easier to roll it back in case.